pkg/util/port/portlist/netstat_macos.go

```diff
@@ -0,0 +1,148 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build darwin
```

Comment:
FYI if you rename the file `netstat_darwin.go` then you can get rid of the build tag (the build tag is added implicitly)